tutorials/roofit/rf204a_extendedLikelihood.py

```diff
@@ -28,7 +28,7 @@
 # Build Chebychev polynomial pdf
 a0 = ROOT.RooRealVar("a0", "a0", 0.5, 0.0, 1.0)
 a1 = ROOT.RooRealVar("a1", "a1", 0.2, 0.0, 1.0)
-bkg = ROOT.RooChebychev("bkg", "Background", x, ROOT.RooArgSet(a0, a1))
+bkg = ROOT.RooChebychev("bkg", "Background", x, [a0, a1])
```

Comment:
Why a list here?

Reply:
It was a mistake that a RooArgSet was passed before, the constructor takes a RooChebychev: https://root.cern.ch/doc/master/classRooChebychev.html#ac4ca6602995755e1818af160c65f3aed

The code just worked before because of the implicit conversion from RooArtSet to RooArgList. But It should be a RooArgList, and our convention in PyROOT is to replace RooArgList with Python lists.